documentation/airport-guides/lkpr.md

```diff
@@ -0,0 +1,66 @@
+### About Prague
+
+Mezinárodní letiště Václava Havla Praha (Václav Havel Airport Prague) (ICAO: LKPR, IATA: PRG) is the main and biggest airport in the Czech Republic, acting as the hub for:
+- Czech Airlines (ICAO: CSA, callsign: CSA)
+- Travel Service and their subsidiary SmartWings (ICAO: TVS, callsign: SKYTRAVEL)
+- Ryanair (ICAO: RYR, callsign: RYANAIR)
+- Wizz Air (ICAO: WZZ, callsign: WIZZAIR),
+
+And a focus city for: 
+- easyJet (ICAO: EZY, callsign: EASY)
+- Vueling (ICAO: VLG, callsign: VUELING)
+
+It serves the capital city and most of the area west of Prague. In the east it is helped by Brno Tuřany (LKTB), Pardubice Airport (LKPD) and Ostrava Mošnov (LKMT). It is located 7NM (13 km) to the North-West of the city centre. 
+
+The Czech civil aviation history started at Prague Kbely Airport in 1918, but it's capacity was insufficient by the mid 1930s the Czech government decided to build a new airport in Prague-Ruzyně. Throughout the years, the airport went through many developmental changes, mainly the construction of runways 04/22, 06/24 and 12/30. 
```

Comment:
its capacity (it's = it is)

replace with this:
The Czech civil aviation history started at Prague Kbely Airport in 1918, but as its capacity was insufficient by the mid-1930s, the Czech government decided to build a new airport in Prague-Ruzyně. Throughout the years, the airport went through many developmental changes, mainly the construction of runways 04/22, 06/24 and 12/30.